public/_redirects

```diff
@@ -4,33 +4,33 @@
 # Redirect default Netlify subdomain to primary domain
 https://eslint.netlify.com/* https://eslint.org/:splat 301!
 
-/docs                               /
-/docs/user-guide/rules              /docs/rules
-/cla                                https://cla.js.foundation/eslint/eslint
-
-/docs/0.24.1/command-line-interface /docs/user-guide/command-line-interface
-/docs/0.24.1/configuring            /docs/user-guide/configuring
-/docs/0.24.1/integrations           /docs/integrations
-/docs/0.24.1/user-guide/rules       /docs/rules
-
-/docs/1.0.0/command-line-interface  /docs/user-guide/command-line-interface
-/docs/1.0.0/configuring             /docs/user-guide/configuring
-/docs/1.0.0/integrations            /docs/integrations
-/docs/1.0.0/user-guide/rules        /docs/rules
-
-/docs/1.10.3/command-line-interface /docs/user-guide/command-line-interface
-/docs/1.10.3/configuring            /docs/user-guide/configuring
-/docs/1.10.3/integrations           /docs/integrations
-/docs/1.10.3/user-guide/rules       /docs/rules
-
-/docs/2.0.0/command-line-interface  /docs/user-guide/command-line-interface
-/docs/2.0.0/configuring             /docs/user-guide/configuring
-/docs/2.0.0/integrations            /docs/integrations
-/docs/2.0.0/user-guide/rules        /docs/rules
-
-/docs/2.13.1/command-line-interface /docs/user-guide/command-line-interface
-/docs/2.13.1/configuring            /docs/user-guide/configuring
-/docs/2.13.1/integrations           /docs/integrations
-/docs/2.13.1/user-guide/rules       /docs/rules
-
-/docs/3.0.0/user-guide/rules        /docs/rules
+/docs                               / 301!
+/docs/user-guide/rules              /docs/rules 301!
+/cla                                https://cla.js.foundation/eslint/eslint 301!
```

Comment:
This doesn’t seem like the right URL anymore?

Reply:
And maybe should be a 302 so we can change this whenever we need to?

Reply:
A `302` makes sense to me!

Do you know what the URL should be? I see that the link in the checks below points to `https://cla.js.foundation/eslint/website?pullRequest={PR_number}`. Surprised that it's not hosted at `openjsf.org`.

Reply:
I have no idea, a quick search didn’t reveal anything. I suppose we can leave it as-is for now, just with a 302

Reply:
Sounds good. I'll follow up with the OpenJSF to confirm what we should be using.